dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/model/dataRequest/StoredDataRequestMessageObject.kt

```diff
@@ -0,0 +1,17 @@
+package org.dataland.datalandcommunitymanager.model.dataRequest
+
+data class StoredDataRequestMessageObject(
+
+    /**
+     * Contains info about a stored message object on Dataland.
+     * @param contactList  a list of strings which include all contact (mail) details
+     * @param message a string of all messages which were created during the life cycle
+     * @param updateTimestamp the date when the data request has been modified the last time
+     */
+
+    var contactList: List<String>? = null,
+
+    val message: String? = null,
+
+    val updateTimestamp: Long? = null,
```

Comment:
Was just wondering because we have now here updateTimestamp and in the StoredDataRequest we use lastModifiedDate. I think we should use same wording, if the content is from a format point of view identical (for example if both hold epochtimes)

Reply:
done